src/defaults.js

```diff
@@ -283,7 +283,9 @@ function translate (key, variables) {
 /* Default options when initializing JSON Editor */
 const options = {
   upload,
-  prompt_before_delete: true
+  prompt_before_delete: true,
+  useDefault: true,
+  maxRecursions: 99999
```

Comment:
The default value should be undefined by default and so recursion block should be performed only if this option has been set. Remove it from here but remember to document it. (documentation must be added also for the useDefault option)